### PR TITLE
Update crates

### PR DESCRIPTION
**Pull request by alatiera**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -37,18 +37,18 @@ pw = {package= "pipewire", version = "0.5", optional = true}
 serde = {version = "1.0", features = ["derive"]}
 serde_repr = "0.1"
 rand = {version = "0.8", default-features = false}
-zbus = {version = "3.1", features = ["url"]}
+zbus = { version = "3.6", features = ["url"] }
```
Comment (A6GibKm) [inline Cargo.toml:40]: This changes the style of the file.

Comment (alatiera) [reply Cargo.toml:40]: Yea that was intentional for all the lines touched, the spaces is what most of formatting tools default too as well

Comment (A6GibKm) [reply Cargo.toml:40]: I mean, if you want to go this route modify it for all crates on a single commit.

Comment (alatiera) [reply Cargo.toml]: meh, let's keep it as is then

**Comment by bilelmoussaoui** [comment]
I actually have these changes locally + some other cleanups, so I would rather wait until I push them first.

**Comment by bilelmoussaoui** [comment]
No longer needed

**Pull request closed without merging**